Open-Cap-Table-Coalition/Open-Cap-Format-OCF
# Quickstart zipfile will contain required initial_shares property

/kind bug

#### What this PR does / why we need it:
- The quickstart tutorial fails validation when downloaded because of schema changes.

#### Which issue(s) this PR fixes:
Fixes #375 


### How to Test:
- unzip the file located at `docs/tutorials/quickstart/issuance_aperture_science_inc.ocf.zip` 
- run the validator on the unzipped direcotry: `node ./utils/validate.mjs validate-ocf-directory -v -p ~/path/to/unzipped-ocf-zip -t`
- Observe the validator passing
